src/KafkaFlow.Retry.API/Handlers/GetItemsHandler.cs

```diff
@@ -29,9 +31,11 @@ internal class GetItemsHandler : RetryRequestHandlerBase
             this.retryDurableQueueRepositoryProvider = retryDurableQueueRepositoryProvider;
             this.getItemsRequestDtoReader = getItemsRequestDtoReader;
             this.getItemsResponseDtoAdapter = getItemsResponseDtoAdapter;
+            var extendedPath = string.IsNullOrEmpty(endpointPrefix) ? ItemsResource : endpointPrefix.ExtendResourcePath(ItemsResource);
```

Comment:
This logic should be handled by the parent class. Let's create a constructor in the `RetryRequestHandlerBase` accepting the parameter `endpointPrefix`, so then we can move this logic to there and set the `base.ResourcePath` with or without the `endpointPrefix`. The code made on the overridden property `ResourcePath` in this class should be reverted.

Apply the same change for the other 2 handlers.

Reply:
Instead of reverting, perhaps also send to the base class the resource also? (in this case `ItemsResource`)

You probably wonder what is the advantage? The `ResourcePath` would be pre-calculated and never calculated again. If you revert the code, the instruction `base.ResourcePath.ExtendResourcePath(extendedPath)` will be executed every time you access `ResourcePath` property. Does this make sense? It is a matter of performing a very small performance optimization here.

Reply:
It makes total sense :)